### ISSUE TITLE
Error opening vim without a file

### ISSUE THREAD

**Issue by dylan-chong** [ntpeters/vim-better-whitespace#86]
```
$ vim

Error detected while processing function <SNR>50_SetupAutoCommands[37]..<SNR>50_ShouldStripWhitespace:
line    3:
E121: Undefined variable: b:better_whitespace_enabled
E15: Invalid expression: b:better_whitespace_enabled < 0
```

**Comment by dylan-chong** [comment]
Looks like solved by #85

**Comment by ntpeters** [comment]
Confirmed fixed in #85
